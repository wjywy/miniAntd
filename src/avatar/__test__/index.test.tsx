import { render } from '@testing-library/react';
import React from 'react';
import Avatar from '../avatar';
describe('Avatar', () => {
  it('should render correctly', () => {
    const { getByTestId } = render(
      <div data-testid="Avatar">
        <Avatar text="hello"></Avatar>
        <Avatar text="wow"></Avatar>
        <Avatar
          style={{ color: '#f56a00', backgroundColor: '#fde3cf' }}
          text="chessui"
        ></Avatar>
      </div>,
    );
    expect(getByTestId('Avatar')).toMatchSnapshot();
  });
});
