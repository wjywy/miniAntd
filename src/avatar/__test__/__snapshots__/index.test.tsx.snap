// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render correctly 1`] = `
<div
  data-testid="Avatar"
>
  <div
    class="avatar avatar-circle l-Avatar-md"
  >
    <span
      style="font-size: 15.5px;"
    >
      hello
    </span>
  </div>
  <div
    class="avatar avatar-circle l-Avatar-md"
  >
    <span
      style="font-size: 17.3px;"
    >
      wow
    </span>
  </div>
  <div
    class="avatar avatar-circle l-Avatar-md"
    style="color: rgb(245, 106, 0); background-color: rgb(253, 227, 207);"
  >
    <span
      style="font-size: 13.7px;"
    >
      chessui
    </span>
  </div>
</div>
`;
